canonical-web-and-design/snapcraft.io
# Setup typescript for brandstore

## Done
Converted the settings view and related tests to TypeScript

## QA
- Go to https://snapcraft-io-3782.demos.haus/admin/test1LaNg1xi6eonae5R/settings
- Check that the view still works as before (compare to live)

## Issue
Fixes #3767
